step3/dataStores/Storage.js

```diff
@@ -0,0 +1,51 @@
+const uuidv4 = require('uuid/v4');
+
+class Storage {
+
+	constructor () {
+		this.messages = [];
+		this.users = [];
+	}
+
+	saveMessage(username, body) {
+		const message = {
+			username: username,
+			body: body,
+			timestamp: Date.now(),
+			id: uuidv4()
+		};
+		this.messages.push(message);
+		return message;
+	}
+
+	getMessages() {
+		return this.messages;
+	}
+
+	connect(username) {
```

Comment:
W sumie na poziomie storage to powinno mieć nazwy adekwatne do poziomu abstrakcji, np. "dodaj użytkownika", "usuń użytkownika" itp. Czepialstwo. 8)